src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,9 +7,21 @@
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_NRIC = new Prefix("i/");
```

Comment:
Just to clarify, `NRIC` gets parsed in and added to the `Person` but it does not get written to the database yet right?

Reply:
When `addHealthWorker` / `addCommand` is called

Reply:
Yes that is correct! The storage portion for writing has not been changed yet, hence it does not include the NRIC field.